atd-toolbox/read_replica_proxy/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  haproxy:
+    build: .
+    restart: always
```

Comment:
This is what makes this service persist after reboot. It's our system's job to get the docker service running, which then takes care of this for us.